broker/broker_utils/requirements.txt

```diff
@@ -0,0 +1,11 @@
+apache-beam[gcp]==2.25.0
```

Comment:
Document pinned version

Reply:
For posterity, referring to my comment [here](https://github.com/mwvgroup/Pitt-Google-Broker/pull/60#issuecomment-890220265). I will continue to work on these versioning issues (cleanup + document).